usr/lib/cca_stdll/cca_specific.c

```diff
@@ -1650,6 +1650,8 @@ CK_RV token_create_ec_keypair(TEMPLATE * publ_tmpl,
     CK_BYTE q[CCATOK_EC_MAX_Q_LEN];
```

Comment:
In ica_specific.c, line 3888 you increased the q_array by one:
 CK_BYTE q_array[1 + ICATOK_EC_MAX_Q_LEN];

Reply:
Right, but the CCA key token already contains the q value including the format byte. CCATOK_EC_MAX_Q_LEN is defined as 133, i.e. 1 + 2 * 66.